packages/jest/package.json

```diff
@@ -39,9 +39,9 @@
     "ci": "./scripts/ci.sh"
   },
   "dependencies": {
-    "@angular-devkit/architect": ">=0.1000.0 < 0.1100.0",
-    "@angular-devkit/core": "^10.0.0",
-    "jest-preset-angular": "^8.2.1",
+    "@angular-devkit/architect": ">=0.1100.0 < 0.1200.0",
+    "@angular-devkit/core": "^11.0.0",
+    "jest-preset-angular": "^8.3.2",
```

Comment:
I suppose it closes #854 ?

Reply:
Yes, it does. I have mentioned it in the commit message but forgot to mention it in the PR.